packages/nocodb/src/models/Column.ts

```
@@ -1347,7 +1361,9 @@ export default class Column<T = any> implements ColumnType {
       prepareForResponse(updateObj),
     );
 
-    await this.insertColOption(context, column, colId, ncMeta);
+    // insert new col options only if existing colOption meta is deleted
+    if (insertColOpt)
+      await this.insertColOption(context, column, colId, ncMeta);
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)